### PR TITLE
Improve default light and dark themes

### PR DESCRIPTION
**Pull request by ghost**
Improve default light and dark themes as per the discord wish.
Includes highlights matching PolyMC's branding
Screenshot_20220930_201741
Screenshot_20220930_203336

### CODE REVIEW

File: launcher/ui/themes/BrightTheme.cpp [hunk 1]
```diff
@@ -18,19 +18,19 @@ bool BrightTheme::hasColorScheme()
...
-    brightPalette.setColor(QPalette::ButtonText, QColor(49,54,59));
+    brightPalette.setColor(QPalette::Window, QColor(255,255,255));
+    brightPalette.setColor(QPalette::WindowText, QColor(17,17,17));
+    brightPalette.setColor(QPalette::Base, QColor(250,250,250));
+    brightPalette.setColor(QPalette::AlternateBase, QColor(246,246,246));
+    brightPalette.setColor(QPalette::ToolTipBase, QColor(17,17,17));
+    brightPalette.setColor(QPalette::ToolTipText, QColor(255,255,255));
+    brightPalette.setColor(QPalette::Text,  Qt::black);
+    brightPalette.setColor(QPalette::Button, QColor(249,249,249));
+    brightPalette.setColor(QPalette::ButtonText, Qt::black));
```
Comment (TayouVR) [inline launcher/ui/themes/BrightTheme.cpp:29]: as the CI builds state, you forgot a bracket there
```suggestion
    brightPalette.setColor(QPalette::ButtonText, Qt::black);
```

Comment (TayouVR) [inline launcher/ui/themes/BrightTheme.cpp]: as the CI builds state, you forgot a bracket there
```suggestion
    brightPalette.setColor(QPalette::ButtonText, Qt::black);
```

**Review by TayouVR — CHANGES_REQUESTED** [review]

File: launcher/ui/themes/BrightTheme.cpp [hunk 2]
```diff
@@ -18,19 +18,19 @@ bool BrightTheme::hasColorScheme()
...
-    brightPalette.setColor(QPalette::AlternateBase, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ToolTipBase, QColor(49,54,59));
-    brightPalette.setColor(QPalette::ToolTipText, QColor(239,240,241));
-    brightPalette.setColor(QPalette::Text,  QColor(49,54,59));
-    brightPalette.setColor(QPalette::Button, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ButtonText, QColor(49,54,59));
+    brightPalette.setColor(QPalette::Window, QColor(255,255,255));
+    brightPalette.setColor(QPalette::WindowText, QColor(17,17,17));
+    brightPalette.setColor(QPalette::Base, QColor(250,250,250));
+    brightPalette.setColor(QPalette::AlternateBase, QColor(246,246,246));
```
Comment (flowln) [inline launcher/ui/themes/BrightTheme.cpp]: i feel like this is too bright for an alternative base, i can barely make out the difference between them in a list...
before: 
image
after:
image

File: launcher/ui/themes/DarkTheme.cpp [hunk 3]
```diff
@@ -18,18 +18,18 @@ bool DarkTheme::hasColorScheme()
 QPalette DarkTheme::colorScheme()
 {
     QPalette darkPalette;
-    darkPalette.setColor(QPalette::Window, QColor(49,54,59));
+    darkPalette.setColor(QPalette::Window, QColor(49,49,49));
     darkPalette.setColor(QPalette::WindowText, Qt::white);
-    darkPalette.setColor(QPalette::Base, QColor(35,38,41));
-    darkPalette.setColor(QPalette::AlternateBase, QColor(49,54,59));
+    darkPalette.setColor(QPalette::Base, QColor(34,34,34));
+    darkPalette.setColor(QPalette::AlternateBase, QColor(38,38,38));
```
Comment (flowln) [inline launcher/ui/themes/DarkTheme.cpp]: similar to the bright theme, this alternating color is too close to the base color imo
before:
image
after:
image

**Review by flowln — COMMENTED** [review]
i think the alternate base colors are the only major problem i see with this. I'm also not super fan of the dark theme's change that removed some of the blue and green from it, but its not objectively bad by any means

else, it looks alright to me!

**Review by Scrumplex — APPROVED** [review]
I think this looks pretty good. Maybe we can even make this the default theme?

**Comment by TayouVR** [comment]
> I think this looks pretty good. Maybe we can even make this the default theme?

on all platforms?
Or on windows specifically?

I think it'd make sense to keep stuff system theme for the most part. On windows due to the lack of dark theme support I could see this being useful, yea

**Comment by TayouVR** [comment]
actually; can't we apply custom colors to the windowsvista theme? Add our own windows dark theme?

**Comment by Scrumplex** [comment]
I thought about making it the *default* on all platforms. If someone cares about it, they can always change it back to System. Maybe we can just make it part of the setup wizard.

**Comment by TayouVR** [comment]
> Maybe we can just make it part of the setup wizard.

I think that would be best, many other apps do that.
I wonder if there is a way to preview the theme kinda like this:
image

Ofc we could fiddle around with making our own widget for something like that, but there's gotta be something someone has already made.... I hope...

**Comment by DioEgizio** [comment]
> I think this looks pretty good. Maybe we can even make this the default theme?

i think it's an improvement, but it's still too fusion-ish to default it IMO, especially on buttons, or on anything widget-related

image
image
image


also on linux using your system theme is a better choice most of the times IMO

**Comment by ghost** [comment]
I'd only default it on windows, leave mac/linux to native

**Comment by ghost** [comment]
Ideally there'd be an "automatic" theme option, which would choose the default light or dark theme on windows and native theme on everything else

**Review by timoreo22 — APPROVED** [review]

**Review by flowln — APPROVED** [review]

**Pull request merged**